get-started/get-started.adoc

```diff
@@ -9,6 +9,13 @@
 Taking the first steps with Neo4j?
 Read our introductory tutorials and developer guides that cover all areas of your development work.
 
+These pages were written to guide you through much of your initial learning for Neo4j and give you resources to continue your education.
+We cover a variety of topics - from graph database concepts to building applications interacting with Neo4j to running Neo4j in production.
+
+There are a variety of materials in different formats for consumption.
+Videos, text, examples, and code are just some of the ways we deliver the information to encourage you and aid all types of learning styles.
```

Comment:
edit: "Videos, text, examples, and code are just some of the **formats in which** we deliver ~~the~~ information to encourage you and aid all ~~types of~~ learning styles.